data/dbHelpers.js

```diff
@@ -0,0 +1,33 @@
+const knex = require('knex');
+const dbConfig = require('../knexfile');
+const db = knex(dbConfig.development);
+
+module.exports = {
+  getDishes: () => {
+    return db('dishes');
+  },
+
+  getRecipes: () => {
+    return db('recipes');
+  },
+
+  getDish: (id) => {
+    return db('dishes')
+            .innerJoin('recipes', 'dish_id', 'dishes.id')
+            .where('dish_id', id);
+  },
+
+  addDish: (newDish) => {
+    return db('dishes').insert(newDish);
+  },
+
+  addRecipe: (newRecipe) => {
+    return db('recipes').insert(newRecipe);
+  },
+
+  shoppingList: (recipe) => {
+    return db('ingredients')
+            .innerJoin('recipes', 'recipe_id', 'recipes.id')
+            .where('recipes.recipe', recipe);
```

Comment:
A better name for your `recipe` column might be `name` so that you don't have this possibly confusing `recipes.recipe` chain.